src/client/gui/pubspec.yaml

```diff
@@ -26,6 +26,7 @@ dependencies:
   grpc: ^4.0.1
   hotkey_manager: ^0.2.3
   intl: ^0.19.0
+  local_notifier: ^0.1.6
```

Comment:
I see that this plugin requires `libnotify-dev` in order to build on Linux. Even though we don't need its functionality on Linux, it's still needed in order to build. So you'll need to add `libnotify-dev` to our `debian/control`. Surprisingly, I see that the snap builds fine, so something must be bringing in that library.